core/helpers/EEH_Sideloader.helper.php

```diff
@@ -69,20 +69,20 @@ private function _init($init)
 
         $props = array_merge($defaults, $init);
 
-        foreach ($props as $propery => $val) {
-            $setter = 'set' . $propery;
+        foreach ($props as $property => $val) {
+            $setter = 'set' . $property;
```

Comment:
BAHAHA how long did we both look at that without noticing the typo? :facepalm:

Reply:
Just a minute.... or 3 :)